### PR TITLE
bower prune

### PR DESCRIPTION
**Pull request by wheresrhys**
@matthew-andrews @i-like-robots

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -11,7 +11,7 @@ SHELL := /bin/bash
 # Some handy utilities
 GLOB = git ls-files -z $1 | tr '\0' '\n' | xargs -I {} find {} ! -type l
 NPM_INSTALL = npm prune --production=false && npm install
-BOWER_INSTALL = bower install --config.registry.search=http://registry.origami.ft.com --config.registry.search=https://bower.herokuapp.com
+BOWER_INSTALL = bower prune && bower install --config.registry.search=http://registry.origami.ft.com --config.registry.search=https://bower.herokuapp.com
```
Comment (matthew-andrews) [inline Makefile:14]: does it need the registry?

Comment (wheresrhys) [inline Makefile:14]: In theory all the information it needs is already there in each component's `.bower.json`, though I don't know this. I'll test it locally

Comment (matthew-andrews) [inline Makefile:14]: yes it probably doens't

Comment (wheresrhys) [inline Makefile:14]: I renamed .bowerrc to .bowerrcccc locally. bower prune succeeded, but then bower install failed, so it looks like prune is a wholly local process

**Comment by matthew-andrews** [comment]
+1

**Pull request merged**